Source/Core/Core/ARDecrypt.cpp

```diff
@@ -209,45 +211,43 @@ constexpr Seeds genseeds = [] {
   return seeds;
 }();
 
-static void getcode(const u32* src, u32* addr, u32* val)
+static std::pair<u32, u32> GetCode(const u32* src)
 {
-  *addr = Common::swap32(src[0]);
-  *val = Common::swap32(src[1]);
+  return {Common::swap32(src[0]), Common::swap32(src[1])};
 }
 
-static void setcode(u32* dst, u32 addr, u32 val)
+static void SetCode(u32* dst, u32 addr, u32 val)
```

Comment:
These parameters could be const, and likewise for various other ones throughout the file (excluding `std::span` since `const` doesn't do anything for them).